src/types/interface.types.ts

```diff
@@ -0,0 +1,5 @@
+import { WalletKind } from "./enum.types";
+
+export interface Wallet {
+	hasAlgoSigner: WalletKind;
```

Comment:
I suggested to rename the `hasAlgoSigner`. Please re read my comment or respond if you have a better idea.